exe/hooks/rthook.py

```diff
@@ -12,3 +12,10 @@
     # See https://bugreports.qt.io/browse/QTBUG-87014
     # and https://github.com/natcap/invest/issues/384
     os.environ['QT_MAC_WANTS_LAYER'] = '1'
+
+    # Rtree will look in this directory first for libspatialindex_c.dylib.
+    # In response to issues with github mac binary builds:
+    # https://github.com/natcap/invest/issues/594
+    # sys._MEIPASS is the path to where the pyinstalleri entrypoint bundle
```

Comment:
```suggestion
    # sys._MEIPASS is the path to where the pyinstaller entrypoint bundle
```

Reply:
Trying out this suggestion feature for a tiny typo!

Reply:
Neat!  I just applied your suggestion and it gave me a chance to edit the commit message (great chance to add the issue number).

Reply:
Looks like the commit shows up as having been `Co-authored-by: Emily Soth <43770515+emlys@users.noreply.github.com>` too!  Very neat.